cmd/certsuite/run/run.go

```diff
@@ -52,6 +52,8 @@ func NewCommand() *cobra.Command {
 	runCmd.PersistentFlags().String("connect-api-base-url", "", "Base URL for Red Hat Connect API")
 	runCmd.PersistentFlags().String("connect-api-proxy-url", "", "Proxy URL for Red Hat Connect API")
 	runCmd.PersistentFlags().String("connect-api-proxy-port", "", "Proxy port for Red Hat Connect API")
+	// When enabled, certsuite will deploy a uniquely named probe daemonset per run
+	runCmd.PersistentFlags().Bool("unique-probe-name", false, "Deploy a uniquely named probe daemonset per run")
```

Comment:
Wouldn't we also need to use cobra to mark this mutually-exclusive to another flag, so that this is truly never used by partners, and only for development?